src/test/regress/input/appendonly.source

```diff
@@ -811,3 +811,29 @@ insert into fix_ao_truncate_last_sequence select 1, 1 from generate_series(1, 5)
 select count(*) from fix_ao_truncate_last_sequence;
 abort;
 
+-- check reporting of missing attributes for AO table
+begin;
+create table test_reporting (id int) with (appendonly=true) distributed by (id);
+select xmin, * from test_reporting;
+abort;
+
+-- check reporting of missing attributes for AO table
+begin;
+create table test_reporting (id int) with (appendonly=true) distributed by (id);
+explain select xmin, * from test_reporting;
```

Comment:
why only one explain? may be do explain for all 4 testing attributes?

Reply:
adding testing columns to explain won't change anything - error will happen at first one and only one will be reported

Reply:
I think, we may explaun only checks

Reply:
please elaborate

Reply:
I suggest leaving only tests with an explain.

Reply:
If you mean that some tests are SELECTs and some are EXPLAIN SELECT then we relying on the very basic fact that EXPLAIN guaranteed to be absolutely the same in every detail as corresponding SELECT. Perhaps it's true, although I've got no this intuition yet.
From the other side, what's wrong to use both SELECT's and EXPLAIN's in the same test?